addons/resource.language.en_gb/resources/strings.po

```diff
@@ -22485,3 +22485,13 @@ msgstr ""
 msgctxt "#39153"
 msgid "Windowing system:"
 msgstr ""
+
+#: system/settings/settings.xml
```

Comment:
Please add a verbal description of the use case for the new string, to give the translators the chance to understand and find the best translation.

Syntax: `#. descriptive text`